src/authentication.ts

```diff
@@ -163,6 +163,12 @@ export interface IAuthenticationAuthorityOptions {
   onForgotPasswordEmailRequest: (user: any, token: string, req?: Request, res?: Response) => Promise<any>;
 
   /**
+   * Called when the user requests a "change password" email.
+   * @param user
+   */
+  onChangePasswordEmailRequest: (user: any, req?: Request, res?: Response) => Promise<any>;
```

Comment:
This should be optional (`onChangePasswordEmailRequest?:`) - unlike `onUserCreated`, or `onResendEmailConfirmation`, `onChangePasswordEmailRequest` is an optional notification. It's up to the implementor to decide if he/she wants to send an email (or whatever).